content/fundamentals/dynamic-modules.md

```diff
@@ -261,6 +261,18 @@ One final note: for simplicity we used a string-based injection token (`'CONFIG_
 export const CONFIG_OPTIONS = 'CONFIG_OPTIONS';
 ```
 
+### Community Guidelines
+
+You may have seen the use for methods like `forRoot`, `register`, and `forFeature` around some of the `@nestjs/` packages and may be wondering what the difference for all of these methods are. There is no hard rule about this, but the `@nestjs/` packages try to follow these guidelines:
+
+When creating a module with `register` you are expecting to configuring a dynamic module for this module only with a specific configuration. For example, with Nest's `@nestjs/axios`: `HttpModule.register({{ '{' }} baseUrl: 'someUrl' {{ '}' }})`. If, in another module you use `HttpModule.register({{ '{' }} baseUrl: 'somewhere else' {{ '}' }})`, it will have the different configuration. You can do this for as many modules as you want.
```

Comment:
```suggestion
When creating a module with `register`, you are expecting to configure a dynamic module with a specific configuration for use only by the calling module . For example, with Nest's `@nestjs/axios`: `HttpModule.register({{ '{' }} baseUrl: 'someUrl' {{ '}' }})`. If, in another module you use `HttpModule.register({{ '{' }} baseUrl: 'somewhere else' {{ '}' }})`, it will have the different configuration. You can do this for as many modules as you want.
```